package.json

```diff
@@ -18,7 +18,9 @@
     "Richard Chamberlain <richard_chamberlain@uk.ibm.com> (https://github.com/rnchamberlain)"
   ],
   "scripts": {
-    "test": "tap test/test*.js"
+    "test": "npm run lint && npm run tap",
+    "tap": "tap --timeout 10 test/test*.js",
+    "lint": "python cpplint.py **/*.cc **/*.h"
```

Comment:
Why have that as pretest? It means that you can't just do `npm run lint` to lint your code.